pysph/sph/acceleration_eval_cython.mako

```diff
@@ -135,6 +150,7 @@ with profile_ctx("nnps.update"):
 % if group.post:
 ${indent(helper.get_post_call(group), 0)}
```

Comment:
You have not covered the Group.post here.

Reply:
I've added the `with profile_ctx(...)` in the cython helper file as the pre and post are pure python calls.